# hardcoded paths for scicore cluster and goolf-1.4.10 toolchain
# you should adapt this to your environment and toolchain

# by now no other solution. Reported to developers. Maybe this is fixed in coming releases
# https://github.com/I2PC/scipion/issues/357#issuecomment-193453973

diff -ru scipion.orig/config/templates/scipion.template scipion/config/templates/scipion.template
--- scipion.orig/config/templates/scipion.template	2016-02-22 14:44:24.000000000 +0100
+++ scipion/config/templates/scipion.template	2016-03-07 21:47:37.273010000 +0100
@@ -65,14 +65,14 @@
 MPI_CXX = mpiCC
 MPI_LINKERFORPROGRAMS = mpiCC
 MPI_LIB = mpi
-MPI_LIBDIR = /usr/lib64/mpi/gcc/openmpi/lib
-MPI_INCLUDE = /usr/lib64/mpi/gcc/openmpi/include
-MPI_BINDIR = /usr/lib64/mpi/gcc/openmpi/bin
+MPI_LIBDIR = /scicore/soft/apps/OpenMPI/1.6.4-GCC-4.7.2/lib
+MPI_INCLUDE = /scicore/soft/apps/OpenMPI/1.6.4-GCC-4.7.2/include
+MPI_BINDIR = /scicore/soft/apps/OpenMPI/1.6.4-GCC-4.7.2/bin
 
 MATLAB_DIR = /usr/local/MATLAB/R2011a
 
 # Java variables
-JAVA_HOME = /usr/lib64/jvm/java-1.7.0-openjdk-1.7.0
+JAVA_HOME = /scicore/soft/apps/Java/1.7.0_80
 JAVA_BINDIR = %(JAVA_HOME)s/bin
 JAVAC = %(JAVA_BINDIR)s/javac
 JAR = %(JAVA_BINDIR)s/jar
